lib/reducers/app_reducer.dart

```diff
@@ -76,8 +74,3 @@ class AddProblemReducer extends TypedReducer<AppState, AddProblem> {
             state.rebuild((b) => b..problems.add(action.problem)));
 }
 
-class RemoveProblemReducer extends TypedReducer<AppState, RemoveProblem> {
```

Comment:
Just a note to remind us, if we decide we do want RemoveProblem action to leave the reducer in too